app/classes/Transvision/Utils.php

```diff
@@ -364,6 +364,26 @@ public static function logScriptPerformances()
     }
 
     /**
+     * Utility function to put into http header the memory used
+     * by a script and the time needed to generate the page
+     *
+     * @return void
+     */
+    public static function logScriptPerformancesHeader()
+    {
+        if (DEBUG && PERF_CHECK) {
+            $memory = 'Memory peak '
+                      . memory_get_peak_usage(true)
+                      . ' ('
+                      . round((memory_get_peak_usage(true) / (1024 * 1024)), 2)
+                      . 'MB)';
+            $render_time = 'Elapsed time (s) '
+                           . round((microtime(true) - $_SERVER['REQUEST_TIME_FLOAT']), 4);
+            header('Transvision-perf: ' . $memory . ' ; ' . $render_time);
```

Comment:
No need for an extra space before the semicolon.
nit for the description: http header -> HTTP header